tests/main/canonical-livepatch/task.yaml

```diff
@@ -16,4 +16,12 @@ execute: |
     done
 
     echo "And ensure we get the expected status"
-    canonical-livepatch status | MATCH "Machine is not enabled"
+    case "$(uname -r)"; in
```

Comment:
Not sure uname -r is the best way to detect this, but it is good enough for the gce kernel.